oscillatord/monitoring.go

```diff
@@ -231,6 +231,46 @@ type Status struct {
 	Clock      Clock      `json:"clock"`
 }
 
+func (s *Status) JSON() ([]byte, error) {
+	output := struct {
+		Temperature         int64 `json:"ptp.timecard.oscillator.temperature"`
+		FineCtrl            int64 `json:"ptp.timecard.oscillator.fine_ctrl"`
+		CoarseCtrl          int64 `json:"ptp.timecard.oscillator.coarse_ctrl"`
+		Lock                int64 `json:"ptp.timecard.oscillator.lock"`
+		GNSSFixNum          int64 `json:"ptp.timecard.gnss.fix_num"`
+		GNSSFixOk           int64 `json:"ptp.timecard.gnss.fix_ok"`
+		GNSSAntennaPower    int64 `json:"ptp.timecard.gnss.antenna_power"`
+		GNSSAntennaStatus   int64 `json:"ptp.timecard.gnss.antenna_status"`
+		GNSSLSChange        int64 `json:"ptp.timecard.gnss.leap_second_change"`
+		GNSSLeapSeconds     int64 `json:"ptp.timecard.gnss.leap_seconds"`
+		GNSSSatellitesCount int64 `json:"ptp.timecard.gnss.satellites_count"`
+		ClockClass          int64 `json:"ptp.timecard.clock.class"`
+		ClockOffset         int64 `json:"ptp.timecard.clock.offset"`
```

Comment:
I don't think this belongs here. It's super weird that Status parsed from json serializes to completely different json. Also, 'ptp.timecard' prefix is too specific for general purpose library.

Reply:
This is just a copy. Literally move from 1 place to another.
Regarding the location - I am convinced it should be next to the struct definition - not cmd.

Reply:
It's a cli-specific structure, and belongs to the cmd that produces it.

Reply:
@abulimov I can name the JSON function something else. Another reason to move it here is because it's clearly operating on the object instance and should be next to the object.
What if we call it "MonitoringJSON"?